src/request_error.js

```diff
@@ -6,13 +6,11 @@ class RequestError extends Error {
      * constructor
      * @param {string} message
      * @param {PassThrough} response
-     * @param {Error} originalError
      */
-    constructor(message, response, originalError) {
+    constructor(message, response) {
         super(message);
         this.response = response;
```

Comment:
Do we need the full response? It makes the error hard to serialize or log.

Reply:
I would probably pick only headers, body and status. You typically don't any other info.